recipes/injector/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "injector" %}
+{% set version = "0.18.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/injector-{{ version }}.tar.gz
+  sha256: ced88ee14183b9f95b2cb6cdb17bf7382499fad187dee0dace6891874ae4b182
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pypandoc
+    - python
+  run:
+    - python
+    - typing_extensions >=3.7.4  # [py<39]
```

Comment:
I'm trying to remember if we can get rid of the `# [py<39]` selector if the backport module just falls back to the stdlib on when the backport isn't needed. If this is the case, then we can make this `noarch: python`. 

@conda-forge/help-python

Reply:
At least for all the ones that are relevant in our current case, cf. e.g.

https://github.com/python/typing/blob/95d6776c0d07e7b096ca153ed6c5b1b30db738e1/typing_extensions/src_py3/typing_extensions.py#L170-L171

In fact, injector duplicates that work by trying `typing` first:

https://github.com/alecthomas/injector/blob/0edfc5524d56f5003786d5821ba822d7cf900ba7/injector/__init__.py#L43-L46

Reply:
Ah, or maybe the point is that it's nice enough to avoid the dependency, but even nicer to be able to use `noarch` out of the box?

Reply:
I don't understand, it looks like the `typing_extensions` package is a backport of something in >= Python 3.7?

Reply:
@dopplershift: Yep, but it also backports Python 3.9 stuff to Python 3.7 (cf. e.g. [this](https://github.com/alecthomas/injector/blob/0edfc5524d56f5003786d5821ba822d7cf900ba7/injector/__init__.py#L127-L128)).

Reply:
Ah. Ok, given that injector tries both, I'd make it an unconditional dependency in order to keep the recipe noarch. Can also open an issue when the feedstock gets created to remind maintainers to remove the dependency when we're only worried about Python >=3.9.

Reply:
Fine with me, and done; it was included mainly for consistency with the PyPI package.